### PR TITLE
Fix shuffling questions before sending them to frontend.

### LINKED ISSUE #47: Check question shuffling for exam
Seems like frontend still gets questions in ascending order, instead of shuffled.

### PR DESCRIPTION
**Pull request by kasya**
Closes #47

### CODE REVIEW

File: apps/question/views.py [hunk 1]
```diff
@@ -36,8 +37,10 @@ def get(self, request, **kwargs):
         aa.question.id
         for aa in AnswerAttempt.objects.filter(attempt=self.attempt)
     ]
+    questions = list(self.attempt.questions.all())
```
Comment (arkid15r) [inline apps/question/views.py]: Something like
```
questions = self.attempt.questions.exclude(id__in=answered_question_ids)   #  rename `answered_questions`
return Response(QuestionSerializer(random.choice(questions)).data)
```
would look more compact

File: apps/question/views.py [hunk 2]
```diff
@@ -36,8 +37,10 @@ def get(self, request, **kwargs):
         aa.question.id
```
Comment (arkid15r) [inline apps/question/views.py]: `answered_questions` could be extracted to an `ExamAttempt` property

File: apps/question/views.py (hunk 1 above)
Comment (kasya) [reply apps/question/views.py]: Yes, looks much better.

File: apps/question/views.py (hunk 2 above)
Comment (kasya) [reply apps/question/views.py]: Oops, I missed this one. I'll fix it with next commit.

File: apps/exam/models.py [hunk 3]
```diff
@@ -87,6 +112,9 @@ def time_left_seconds(self):
     time_left = end_time - timezone.now()
     return max(int(time_left.total_seconds()), 0)
 
+  def __str__(self):
```
Comment (arkid15r) [inline apps/exam/models.py]: It's okay to keep `__str__` at the top of the class.

**Review by arkid15r — APPROVED** [review]
LGTM

**Pull request merged**